UIKitTests/UIScrollViewTests.swift

```diff
@@ -110,6 +110,42 @@ class UIScrollViewTests: XCTestCase {
         // XXX: set style, test bgr -> does this need to be tested?
     }
 
+    func testIsDecelerating() {
+        // This tests an edge case in iOS behaviour (while also testing some basics along the way)
```

Comment:
"while also testing some basics along the way" is probably not the most maintainable approach. if we have basics to test it'd be best to make basic tests for them :)

Reply:
The test is quite long as is and I find it pretty hard to understand what it's actually testing